### PR TITLE
Update getTables requests to support table filters

### LINKED ISSUE #23: Accept tables parameter for getTables request
db-ex issue: https://github.com/keboola/db-extractor-oracle/issues/75

db-ex PR  https://github.com/keboola/db-extractor-oracle/pull/76

### PR DESCRIPTION
**Pull request by pivnicek**
fixes #23

### CODE REVIEW

File: src/main/java/com/keboola/tableexporter/MetaFetcher.java [hunk 1]
```diff
@@ -199,6 +199,20 @@ private String tableListingQuery() {
...
                 "        AND COLS.COLUMN_NAME = REFCOLS.COLUMN_NAME";
+
+        if (tables.size() > 0) {
+            sql += "\nWHERE ";
+            for (int i = 0; i < tables.size(); i++) {
+                if (i > 0) {
+                    sql += " OR ";
+                }
+                sql += "(";
+                sql += "TABS.TABLE_NAME = '" + tables.get(i).getTableName() + "' AND ";
```
Comment (tomasfejfar) [inline src/main/java/com/keboola/tableexporter/MetaFetcher.java]: Not sure this qualifies as identifier quoting.

Comment (tomasfejfar) [reply src/main/java/com/keboola/tableexporter/MetaFetcher.java]: We can probably ship it like this, but definitely it should be fixed.

File: src/test/resources/getTables/config.json [hunk 2]
```diff
@@ -1,5 +1,6 @@
 {
   "parameters": {
-    "outputFile": "/code/tables.json"
+    "outputFile": "/code/tables.json",
+    "tables": []
```
Comment (tomasfejfar) [inline src/test/resources/getTables/config.json]: is this backwards compatible? Will config without it still work?

File: src/main/java/com/keboola/tableexporter/MetaFetcher.java (hunk 1 above)
Comment (odinuv) [reply src/main/java/com/keboola/tableexporter/MetaFetcher.java]: yeah, should use prepared statement and the whole method would have to return `Statement`

File: src/test/resources/getTables/config.json (hunk 2 above)
Comment (odinuv) [reply src/test/resources/getTables/config.json]: doesn't need to be, cause this interfaces with the php script https://github.com/keboola/db-extractor-oracle/pull/76/files#diff-0470f12f4ee34ff6a4ee76ff8ac10b0bR59

Comment (odinuv) [reply src/test/resources/getTables/config.json]: also it is https://github.com/keboola/java-oracle-exporter/pull/24/files#diff-c69eab5c9c99db4f5b1d03ba7654571aR52

**Pull request merged**